src/scss/components/contact.scss

```diff
@@ -0,0 +1,43 @@
+.contactUs {
+  @extend .container;
```

Comment:
Do not extend the class here. Add it in the JS along with the base class. The class `.container` and the base component class has different responsibilities, so please keep them separate.